### PR TITLE
Add engine panel

### PR DESCRIPTION
**Pull request by kitchoi**
- [x] Added `add_engine_panel` for adding engine to the EngineManager panel in `EngineManagerStandaloneUI` and `EngineManagerMayavi2`
- [x] Added tests

engine_manager_standalone_ui_add_engine

### CODE REVIEW

**Comment by kitchoi** [comment]
Ready for review.

**Comment by kitchoi** [comment]
@dpinte This is ready for review

File: simphony_mayavi/plugins/engine_wrappers/loader.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import sys
+import importlib
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def get_factories(submodules=("lammps_md", "kratos", "openfoam", "jyulb"),
```
Comment (dpinte) [inline simphony_mayavi/plugins/engine_wrappers/loader.py]: I would suggest moving the wrappers tuple as a `DEFAULT_ENGINES` contant and defaulting to `submodules=DEFAULT_ENGINES`

**Comment by dpinte** [comment]
@kitchoi LGTM with a minor comment.

**Comment by kitchoi** [comment]
Thanks!

**Pull request merged**